publications-2018.bib

```diff
@@ -1,11 +1,12 @@
 % Encoding: US-ASCII
 
-@Article{2018_0,
-  author    = {G. Alzetta and D. Arndt and W. Bangerth and V. Boddu and B. Brands and D. Davydov and R. Gassmoeller and T. Heister and L. Heltai and K. Kormann and M. Kronbichler and M. Maier and J.-P. Pelteret and B. Turcksin and D. Wells},
+@Article{dealII90,
   title     = {The \texttt{deal.II} Library, Version 9.0},
+  author    = {G. Alzetta and D. Arndt and W. Bangerth and V. Boddu and B. Brands and D. Davydov and R. Gassmoeller and T. Heister and L. Heltai and K. Kormann and M. Kronbichler and M. Maier and J.-P. Pelteret and B. Turcksin and D. Wells},
```

Comment:
don't see what was actually changed here.

Reply:
title in front of author

Reply:
oh, git makes this very confusing...